pkg/standalone-utils/contracts/StableLPOracle.sol

```diff
@@ -147,67 +158,65 @@ contract StableLPOracle is LPOracleBase {
         // solhint-disable-previous-line var-name-mixedcase
 
         uint256 i;
         int256 den;
 
         // To avoid overflows, we divided f(k) and f'(k) by PT^n. That's why, instead of computing P, P' and alpha,
-        // we change the variables to compute `dPdkDivP = P'/P` and `alphaDivPTn = alpha/(PT^n)`.
+        // we change the variables to compute `PPrimeOverKDividedByP = P'/P` and `alphaDivPTn = alpha/(PT^n)`.
 
         T = 0;
-        // dTdk = T', where T' is the derivative of T with respect to k.
-        dTdk = 0;
-        // dPdkDivP = P'/P, where P' is the derivative of P with respect to k.
-        dPdkDivP = 0;
+        // TPrimeOverK = T', where T' is the derivative of T with respect to k.
+        TPrimeOverK = 0;
+        // PPrimeOverKDividedByP = P'/P, where P' is the derivative of P with respect to k.
+        PPrimeOverKDividedByP = 0;
         // alphaDivPTn = alpha/(PT^n), where alpha is the constant term of the stable invariant.
         alphaDivPTn = b;
         for (i = 0; i < _totalTokens; i++) {
             den = _mulDownInt(k, prices[i]) - a;
             T += _divDownInt(a, den);
-            dTdk -= _divDownInt((prices[i] * a) / den, den);
-            dPdkDivP += _divDownInt(prices[i], _mulDownInt(k, prices[i]) - a);
+            TPrimeOverK -= _divDownInt((prices[i] * a) / den, den);
+            PPrimeOverKDividedByP += _divDownInt(prices[i], _mulDownInt(k, prices[i]) - a);
             alphaDivPTn = (alphaDivPTn * b) / a;
         }
-        T -= _ONE_INT;
+        T -= _POSITIVE_ONE_INT;
 
         // We need to calculate T and alpha first to be able to divide alpha by PT^n.
         for (i = 0; i < _totalTokens; i++) {
             int256 ri = _divDownInt(prices[i], a);
             // P = \prod(k*ri - 1) . So, to divide alpha by PT^n, we can iteratively divide alpha by T and (k*ri - 1).
-            den = _mulDownInt(k, ri) - _ONE_INT;
+            den = _mulDownInt(k, ri) - _POSITIVE_ONE_INT;
             alphaDivPTn = _divDownInt(alphaDivPTn, _mulDownInt(den, T));
         }
     }
 
     function _computeAAndBForPool(IStablePool pool) internal view returns (int256 a, int256 b) {
-        (uint256 amplificationFactor, , ) = pool.getAmplificationParameter();
-        // a = A * n^2n, but A = ampParameter * n^(n-1). So, a = ampParameter * n^(2n)/n^(n-1) = ampParameter * n^(n+1).
-        a = int256((amplificationFactor * (_totalTokens ** (_totalTokens + 1))).divDown(StableMath.AMP_PRECISION));
+        (uint256 amplificationParameter, , ) = pool.getAmplificationParameter();
+        // In the StableMath library, `amplificationParameter = A*n^(n-1)` (For more information, check the
+        // `computeInvariant` natspec of the StableMath library).
+        // a = A * n^2n, but A = ampParameter / n^(n-1). So, a = ampParameter * n^(2n)/n^(n-1) = ampParameter * n^(n+1).
+        a = int256((amplificationParameter * (_totalTokens ** (_totalTokens + 1))).divDown(StableMath.AMP_PRECISION));
```

Comment:
Let's maybe add a small comment explaining why the unsafe cast is actually safe. It should be because we're dividing by AMP_PRECISION.

Also, why is it a FP operation instead of a raw one?

Reply:
When I do an FP division, I scale `a` to 18 decimals. Same with `b`. So, I can use these variables as scaled 18 variables, which is prety convenient when I'm doing divisions like `a/b`, where I need floating points. I'll add a comment about it.